components/script/dom/document.rs

```diff
@@ -5306,7 +5306,7 @@ impl DocumentMethods for Document {
         url: USVString,
         target: DOMString,
         features: DOMString,
-        can_gc: CanGc,
+        _can_gc: CanGc,
```

Comment:
The can_gc variable here is unused, so lmk if it'd be better to simply remove it?

Reply:
Yep, please remove it!

Reply:
I'm not quite sure why the build is failing even after removing it from `Bindings.conf`. Maybe we should just do: `_can_gc`?

Reply:
You're right. It looks like a bug in the code generator, so using the _ prefix for now is fine.

Reply:
Alright, thanks! :)